docs/source/style.rst

```diff
@@ -27,15 +33,15 @@ We adhere to `PEP8 <https://www.python.org/dev/peps/pep-0008/>`_ with the except
 
 We aim to incorporate `PEP 526 <https://www.python.org/dev/peps/pep-0484/>`_ type annotations throughout the library.  See the `Mypy <https://mypy.readthedocs.io/en/stable/>`_ type annotation `cheat sheet <https://mypy.readthedocs.io/en/stable/cheat_sheet_py3.html>`_ for usage examples. Custom types are defined in :mod:`.typing`.
 
-Our coding conventions are based on both the `numpy conventions <https://numpydoc.readthedocs.io/en/latest/format.html#overview>`_ and the `google docstring conventions <https://google.github.io/styleguide/pyguide.html>`_.
+Our coding conventions are based on both the `NumPy conventions <https://numpydoc.readthedocs.io/en/latest/format.html#overview>`_ and the `Google docstring conventions <https://google.github.io/styleguide/pyguide.html>`_.
 
 Unicode variable names are allowed for internal usage (e.g. for Greek characters for mathematical symbols), but not as part of the public interface for functions or methods.
 
 
 Naming
 ------
 
-Naming conventions for the different programming components are as follows:
+Naming conventions adhering to the `Google naming conventions <https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations>`_ for the different programming components are as follows:
```

Comment:
While I appreciate the effort it took to work these docs links into the text, I found them a bit cumbersome to read. It also raises the question: "Do we follow _all_ of the, .e.g., google string conventions, or only the ones we mention specifically?". 

I propose a more formulaic approach to these sections:
"Strings
We follow the Google string conventions. Notably, prefer to use Python f-strings, rather than .format or % syntax. For example, ..."

Reply:
I like this suggestion, but it would be good to keep the links.

Reply:
Agreed, that was my intention but it wasn't clear from my example.